### ISSUE TITLE
Change GPU numbers during training

### ISSUE THREAD

**Issue by YingYingFight** [open-mmlab/mmsegmentation#1776]
Hi,  I read the sentences about GPU numbers in the [mmseg official document ](https://mmsegmentation.readthedocs.io/en/latest/train.html).

> *Important*: The default learning rate in config files is for 4 GPUs and 2 img/gpu (batch size = 4x2 = 8). Equivalently, you may also use 8 GPUs and 1 imgs/gpu since all models using cross-GPU SyncBN.

Does the above sentence mean:
 
**If I hope to change the GPU number from 4 to 8,  I only need to change the image number per GPU,  like:  
change ` data=dict(samples_per_gpu=2)` (4GPUs) to ` data=dict(samples_per_gpu=1)` (8GPUs).** 

**Do I need to modify other configurations to keep the results on 4 GPUs and 8GPUs same?**

Thank you very much for your help!

**Comment by MeowZheng** [comment]
Your understanding is correct, and no need.

Best

**Comment by YingYingFight** [comment]
> Your understanding is correct, and no need.
> 
> Best

I get it. Thank you for your help.

**Comment by jkd2021** [comment]
Hi, guys, I'm also facing a question about the relationship between batchsize and nums of GPUs. Now I'm trying to train the network with a single GPU, whose original setting is "samples per GPU=2, nums of GPUs=4, learning rate=0.01, using syncBN". Am I right that I should change it into "samples per GPU=8, nums of GPUs=1, learning rate=0.01, using BN" (in order to keep iter * lr as a constant) ?
